src/Appwrite/Client.php

```diff
@@ -37,11 +37,11 @@ class Client
      */
     protected array $headers = [
         'content-type' => '',
-        'user-agent' => 'AppwritePHPSDK/12.0.0 ()',
+        'user-agent' => 'AppwritePHPSDK/12.2.0 ()',
         'x-sdk-name'=> 'PHP',
         'x-sdk-platform'=> 'server',
         'x-sdk-language'=> 'php',
-        'x-sdk-version'=> '12.0.0',
+        'x-sdk-version'=> '12.2.0',
```

Comment:
12.1 ?

Reply:
[Latest release was 12.1.0](https://github.com/appwrite/sdk-for-php/releases/tag/12.1.0), must not have had the correct version